specification/dimensions/resourceid.md

```diff
@@ -0,0 +1,30 @@
+# Resource ID
+
+A Resource ID is an identifier assigned to a resource by the provider. The Resource ID is commonly used for cost reporting, analysis and allocation scenarios.
```

Comment:
Do we need to agree on which style guide we're using for writing? For instance, should we use commas before the last item in lists, as shown here?
```suggestion
A Resource ID is an identifier assigned to a resource by the provider. The Resource ID is commonly used for cost reporting, analysis, and allocation scenarios.
```